app/views/users/show.html.erb

```diff
@@ -1,12 +1,16 @@
-<h1><%= @user.name %></h1>
-<ul>
-  <% @user.posts.each do |post| %>
-    <%= link_to user_post_path(@user, post.id) do %>
-      <li>
-        <h3><%= post.title %> __ post.id = <%= post.id %></h3>
-        <p><%= post.text %></p>
-      </li>
+<div class="flex flex-col w-full gap-3">
+  <%= render partial:'shared/user', locals: {user: @user} %>
+  <div class="flex flex-col gap-3 w-full p-4 border border-black border-solid">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <ul class="flex flex-col gap-3 w-full">
+    <p class="only:block hidden">There are no posts yet!</p>
+    <% @user.last_3_posts.each do |post| %>
+      <%= link_to user_post_path(@user, post.id) do %>
+        <%= render partial:'shared/post_highlight', locals: {user: @user, post: post} %>
+      <% end %>
```

Comment:
- [ ] @tiagomarin great work in your views, but you forgot something small, you didn't display the number of comments and likes in the post as it's in the design check the comparison below:

Design Given          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/200675039-74052312-8319-41bc-90e4-9706dd574904.png) |  ![image](https://user-images.githubusercontent.com/61408860/200675087-4b31197e-06c0-4b8f-8cfd-6a462fcce948.png)

Reply:
### :heavy_check_mark: Changes applied

Thanks for pointing that out!